### PR TITLE
[OAP-1823][oap-native-sql] Update installation guide

### LINKED ISSUE #1823: [oap-native-sql][doc] Spark Native SQL Engine installation guide is obsolete and thus broken.

### PR DESCRIPTION
**Pull request by zt-c**
Updated some readme files.
Fixed #1823.

### CODE REVIEW

**Comment by SheltonZChen** [comment]
Fix: #1823

File: oap-native-sql/cpp/compile.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ fi
 mkdir build
 cd build
 cmake ..
-make
+make -j
```
Comment (zhouyuan) [inline oap-native-sql/cpp/compile.sh:15]: this seems making some trouble for Travis, let's using single thread

Comment (SheltonZChen) [reply oap-native-sql/cpp/compile.sh]: Thank you! Reverted this script.

**Comment by zhouyuan** [comment]
thanks @zt-c for the good start, @weiting-chen will help to refine the doc further

**Comment by weiting-chen** [comment]
> thanks @zt-c for the good start, @weiting-chen will help to refine the doc further

Thanks @zt-c , since I am working on the package and release for native SQL, I will submit a new patch with your contribution and some updates.
The patch you submit is really helpful. Thanks again.

**Comment by zhouyuan** [comment]
closing this as Weiting will open up a new PR to refine the doc

**Pull request closed without merging**